skinport/salefeed.py

```diff
@@ -457,6 +510,11 @@ def exterior(self) -> Optional[str]:
         """Optional[:class:`str`]: Returns the exterior of the item."""
         return self._exterior
 
+    @property
+    def quality(self) -> str:
+        """:class:`str`: Returns the quality of the item."""
+        return self._quality
+
```

Comment:
It seems that `quality` is either "Normal", "Souvenir" or "StatTrak™". I think this would justify a new Enum to be implemented for that.